src/triage/risklist/__init__.py

```diff
@@ -0,0 +1,236 @@
+from triage.component.results_schema import upgrade_db, Experiment, ExperimentModel 
+from triage.component.architect.entity_date_table_generators import EntityDateTableGenerator, DEFAULT_ACTIVE_STATE
+from triage.component.architect.features import FeatureGenerator
+from triage.component.architect.feature_group_creator import FeatureGroup
+from triage.component.architect.builders import MatrixBuilder
+from triage.component.architect.planner import Planner
+from triage.component.timechop import Timechop
+from triage.component.catwalk.predictors import Predictor
+from triage.component.catwalk.utils import filename_friendly_hash
+from triage.util.conf import convert_str_to_relativedelta, dt_from_str
+from triage.util.db import scoped_session
+from sqlalchemy import select
+
+from collections import OrderedDict
+import json
+import re
+
+import verboselogs, logging
+logger = verboselogs.VerboseLogger(__name__)
+
+
+def experiment_config_from_model_id(db_engine, model_id):
+    """Get original experiment config from model_id 
+    Args:
+            db_engine (sqlalchemy.db.engine)
+            model_id (int) The id of a given model in the database
+
+    Returns: (dict) experiment config
+    """
+    get_experiment_query = '''select experiments.config
+    from triage_metadata.experiments
+    join triage_metadata.experiment_models using (experiment_hash)
+    join triage_metadata.models using (model_hash)
+    where model_id = %s
+    '''
```

Comment:
Probably overly-cautious, but are we certain this is 1:1?

Reply:
I assumed it was but when I code-dove I found that I misremembered the purpose of experiment_models. That table has all of the experiments that the model is associated with, but Model.built_by_experiment is what actually records the experiment that *built* it.

I'm not sure if the current implementation would end up causing any real problems. If it's associated with an experiment, then that experiment should have the necessary feature and other config to work here. The config that is likely to differ is time config (not used here), or feature config that isn't used in this model, etc.

It's worth changing either way.

Reply:
That's fair -- I can't think of a case where it would make a difference here either, but agree it's probably safer to stick with the built_by_experiment just in case.